### DISCUSSION TITLE
Problem with the test running right after the installation

### DISCUSSION THREAD

**Discussion by shaheidula** [mdolab/dafoam#489] [General]
Hello Prof. Ping, 
When I was trying to run the regression tests at the end of the installation of the latest version, I kept running into the problem as shown in the attached. I tried to reinstall many times but always the same error. 
The error is:: 
pyOptSparse Error: There was an error importing the compiled slsqp module
 
[Regressession.txt](https://github.com/mdolab/dafoam/files/12681062/Regressession.txt)
![5](https://github.com/mdolab/dafoam/assets/66427363/136f64e3-9c76-45a3-981a-84a3c33c67a9)

**Comment by friedenhe** [comment]
It seems that the pyOptSparse was not installed properly. Are you using Ubuntu 20.04?

**Comment by shaheidula** [comment]
Thanks for replying. Yes, I am using Ubuntu 20.04. And I made sure all the packages were installed properly step by step. But When I tried to run the regression test, It failed. I reinstalled it several times and even reinstalled my system once. Still not working.

**Comment by shaheidula** [comment]
Dear Prof. Ping, 
I solved the problem by installing the older version of pyOptSparse. However, when I tried to install TACS, It didn't work out. I guess there is a bug there. Please find the attachment. 

 And there might be a problem with the version of the numpy recommended on the installation list as well, as there is a conflict between the version of numpy recommended and the one which is compatible with the tensorflow-cpu==2.12. 

![2](https://github.com/mdolab/dafoam/assets/66427363/255f9241-ebb0-4167-81f6-8e9aafca3849)

**Reply by friedenhe** [reply]
hmm, not sure why this error showed up. do you need to run aerostructural cases? if not, you can skip the tacs installation.

**Comment by shaheidula** [comment]
Dear Prof. Ping, 
Thanks a lot for your reply. Actually, we need the aerostructural cases.

**Comment by shaheidula** [comment]
Hello Prof. Ping, 
I replaced the TACS version shown in the DAFoam installation list with the latest one on the TACS's Github, and the problem solved and it worked finally. 
Thanks a lot, 
![Uploading 3.PNG…]()
